server/server.js

```diff
@@ -0,0 +1,32 @@
+const path = require('path');
+const express = require('express');
+
+const PORT = 3000;
+const app = express();
+
+app.listen(PORT, console.log(`Listening on PORT: ${PORT}`))
```

Comment:
There's a lot of eslint jank in this file -- mostly semicolons missing and tabs vs. spaces stuff. Can we clean that up?